tee-worker/cli/src/trusted_operation.rs

```diff
@@ -469,6 +393,10 @@ pub(crate) fn send_direct_batch_vc_request(
 				let response: RpcResponse = serde_json::from_str(&response).unwrap();
 				if let Ok(return_value) = RpcReturnValue::from_hex(&response.result) {
 					debug!("successfully decoded rpc response: {:?}", return_value);
+					if ignore_first {
```

Comment:
I could not understand this, Are we deliberately ignoring the first response, if so why?